CHANGELOG.md

```diff
@@ -7,6 +7,18 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 
 ## Unreleased
 
+### New features
+
+#### node
+
+- Client credential grant: for Solid Identity Providers which support it, a client
```

Comment:
Might as well add a link to some documentation on how to set this up for pod.inrupt.com, given that that's going to be the main use case for a while. (Or a direct link to the page to register them.)

Reply:
I'll do this as part of the documentation ticket I'm working on right now, because there is currently nothing documenting the static client registration service.